test/e2e/e2e_test.go

```diff
@@ -169,6 +169,8 @@ func TestChannelChain(t *testing.T) {
 	e2ehelpers.ChannelChainTestHelper(
 		context.Background(),
 		t,
+		// TODO(tommyreddad): eventing 0.22 e2e/helpers has a bug where SubscriptionV1beta1 and SubscriptionV1 are flipped in the ChannelChainTestHelper.
```

Comment:
Create an issue to track this

Reply:
Done. https://github.com/google/knative-gcp/issues/2202